index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    
+    <h1>Hello World we are software devlopers!</h1>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">please fill out the following to provide your feedback ands help improve our service.</p>
+    <form id="survey-form">
+
+      Name: <input type="text" placeholder=" Enter your name"  required> <br>
+      Email: <input type="email" placeholder="Enter your Email"  required><br>
+      Phone Number: <input type="number" placeholder="Please enter your phone number"  required max="99999999" min="0" ><br>
+      <label id="dropdown">How did tou hear about us></label>
+      <select id="select">
+        <option value="Friends">From the friends</option>
+        <option value="internet">From the internet</option>
+        
+      </select>
+  
+      <label for ="radio" id="radio1-lable1">Would you recommond our service to a frind</label>
+      <input type="radio" id="radio1" value="yes">
+      <input type="radio" id="radio1" value="No">
+      <br>
+      <input type="textaera" row="5" col="5">
+      <button type="submit" value="submit">submit</button>
```

Comment:
It would be better to enclose your code inside a  div or section to be very organized.